packages/disputer/src/disputer.js

```diff
@@ -328,31 +322,30 @@ class Disputer {
         continue;
       }
 
-      // Get resolved price request for dispute. `getPrice()` should not fail since the dispute price request must have settled in order for `withdrawLiquidation()`
-      // to be callable.
-      let resolvedPrice = await this.votingContract.methods
-        .getPrice(this.financialContractIdentifier, requestTimestamp)
-        .call({
-          from: this.financialContract.options.address
-        });
-
-      const logResult = {
+      let logResult = {
         tx: receipt.transactionHash,
         caller: receipt.events.LiquidationWithdrawn.returnValues.caller,
-        withdrawalAmount: receipt.events.LiquidationWithdrawn.returnValues.withdrawalAmount,
+        settlementPrice: receipt.events.LiquidationWithdrawn.returnValues.settlementPrice,
```

Comment:
does this work with all previous contracts?

Reply:
Yep I double checked. You actually added this in a while ago (prior to version 1.1.0 even) but we never took advantage of it in the bots. This event change was made in July, a month before we set up our versioning!

https://github.com/UMAprotocol/protocol/pull/1753